autocomplete.js

```diff
@@ -5,12 +5,17 @@ function createAwsAutocompleteFunction(
   methodName,
   outputDataPath,
   [valuePath, labelPath] = [],
+  payloadBuilder = () => ({}),
```

Comment:
It's a function, not an object, thus a verb oriented name fits there better eg. `buildPayload`.